CHANGELOG.md

```diff
@@ -2,66 +2,59 @@
 
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## FAQ
+### What is the purpose of the changelog?
+The purpose of the changelog is for the contributors and maintainers to
+incrementally build the release notes throughout the development process to
+avoid a painful and error-prone process of attempting to compile the release
+notes at release time. On each release the "unreleased" entries of the
+changelog are moved to the appropriate release notes document in the
+`./release-notes` folder.
+
+### Which changes require a CHANGELOG entry?
+Changelogs are intended for _users_. In short, any change that a user of
```

Comment:
i know that i've also used the word "user" before in this context, but i think it's a hard-to-define word here: does this mean the operator of an OpenSearch cluster? or does it mean the developer interacting with OpenSearch through APIs/libraries? or does it mean the (business) end-user interacting with it through OpenSearch Dashboards?

IMHO it should mean _all_ of those, but maybe a short description of what is meant might be helpful to avoid people taking assumptions on the definition of this word and then skipping changelog entries which would've made sense?